config/arc.php

```diff
@@ -91,4 +91,14 @@
     */
     'first_delivery_date' => env('ARC_FIRST_DELIVERY_DATE', '2019-09-26'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | the programme names we are running
+    | eg. "standard", "social prescribing"
+    |--------------------------------------------------------------------------
+     */
+    'programmes' => [
+        'standard',
```

Comment:
Can we have the words starting with capitals? It looks very strange otherwise (imo).